From c63ee4a521e766da6ec5ee1d2058d1ec06216214 Mon Sep 17 00:00:00 2001
From: Namjae Jeon <namjae.jeon@samsung.com>
Date: Wed, 30 Jun 2021 09:39:15 +0900
Subject: [PATCH] ksmbd: change server config string index to enumeration
Git-commit: c63ee4a521e766da6ec5ee1d2058d1ec06216214
Patch-mainline: v5.15-rc1
References: bsn#12
Modified-by-SEL: No


Change server config string index to enumeration.

Reviewed-by: Christoph Hellwig <hch@lst.de>
Signed-off-by: Namjae Jeon <namjae.jeon@samsung.com>
Signed-off-by: Steve French <stfrench@microsoft.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 fs/ksmbd/server.h | 11 ++++++++---
 1 file changed, 8 insertions(+), 3 deletions(-)

diff --git a/fs/ksmbd/server.h b/fs/ksmbd/server.h
index 2fce06e8b833..ac9d932f8c8a 100644
--- a/fs/ksmbd/server.h
+++ b/fs/ksmbd/server.h
@@ -18,9 +18,14 @@ enum {
 	SERVER_STATE_SHUTTING_DOWN,
 };
 
-#define SERVER_CONF_NETBIOS_NAME	0
-#define SERVER_CONF_SERVER_STRING	1
-#define SERVER_CONF_WORK_GROUP		2
+/*
+ * Server global config string index
+ */
+enum {
+	SERVER_CONF_NETBIOS_NAME,
+	SERVER_CONF_SERVER_STRING,
+	SERVER_CONF_WORK_GROUP,
+};
 
 struct ksmbd_server_config {
 	unsigned int		flags;
-- 
2.33.0

